docs/quickstart.rst

```diff
@@ -79,6 +79,28 @@ Example:
 The raw values (not-casted to any type) are still available from their
 usual place on the request (`matchdict`, `GET`, `POST`, `json()`, etc)
 
+If you have ``pyramid_swagger.use_models`` enabled, you can interact with
+models defined in ``#/definitions`` as Python classes instead of dicts.
+
+.. code-block:: python
+
+    @view_config(route_name='add.user)
+    def add_user(request):
+        user = request.swagger_data['user']
```

Comment:
Does it do casting too? Better to use a boolean type to make this crystal clear?